Missionframework/modules/26_cratefiller/fnc/fn_cratefiller_getPlayerInventory.sqf

```diff
@@ -0,0 +1,116 @@
+#include "..\ui\defines.hpp"
+#include "script_component.hpp"
+/*
+    KPLIB_fnc_cratefiller_getPlayerInventory
+
+    File: fn_cratefiller_getPlayerInventory.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2019-04-27
+    Last Update: 2019-05-02
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+    Public: No
+
+    Description:
+        Gets all weapons of the selected player.
+
+    Parameter(s):
+        NONE
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
+// Dialog controls
+private _dialog = findDisplay KPLIB_IDC_CRATEFILLER_DIALOG;
+private _ctrlPlayers = _dialog displayCtrl KPLIB_IDC_CRATEFILLER_COMBOPLAYERS;
+private _ctrlMainWeapon = _dialog displayCtrl KPLIB_IDC_CRATEFILLER_MAINWEAPON;
+private _ctrlHandgun = _dialog displayCtrl KPLIB_IDC_CRATEFILLER_HANDGUN;
+private _ctrlSecondaryWeapon = _dialog displayCtrl KPLIB_IDC_CRATEFILLER_SECONDARYWEAPON;
+
+// Read controls
+private _index = lbCurSel _ctrlPlayers;
+
+// Get the selected player from the namespace
+private _player = (CCGVAR("players", [])) select _index;
+
+// Get the player weapons
+private _mainWeapon = primaryWeapon _player;
+private _handgun = handgunWeapon _player;
+private _secondaryWeapon = secondaryWeapon _player;
+
+// Variables
+private _config = configNull;
+private _picture = "";
+private _tooltip = [];
+private _description = "";
+private _magazine = "";
+private _attachments = "";
+
+// Fill the controls
+if !(_mainWeapon isEqualTo "") then {
```

Comment:
The below code could get some sensible empty lines and more important some comments